summertime/dataset/dataset_loaders.py

```diff
@@ -360,6 +365,170 @@ def is_supported(self, language: str):
             return True
 
 
+class XlsumDataset(SummDataset):
+    """
+    The XLSum dataset - A massively multilingual dataset including 45 languages
+    Contains 1.35 million article-summary pairs from BBC in the following languages:
+
+    """
+
+    dataset_name = "XLSum"
+    description = "A massively multilingual dataset including 45 languages. Contains 1.35 million article-summary pairs from BBC."
+
+    is_query_based = False
+    is_dialogue_based = False
+    is_multi_document = False
+    is_multilingual = True
+
+    huggingface_dataset = True
+    huggingface_page = "https://huggingface.co/datasets/csebuetnlp/xlsum"
+    supported_languages = [
+        "oromo",
+        "french",
+        "amharic",
+        "arabic",
+        "azerbaijani",
+        "bengali",
+        "burmese",
+        "chinese_simplified",
+        "chinese_traditional",
+        "welsh",
+        "english",
+        "kirundi",
+        "gujarati",
+        "hausa",
+        "hindi",
+        "igbo",
+        "indonesian",
+        "japanese",
+        "korean",
+        "kyrgyz",
+        "marathi",
+        "spanish",
+        "scottish_gaelic",
+        "nepali",
+        "pashto",
+        "persian",
+        "pidgin",
+        "portuguese",
+        "punjabi",
+        "russian",
+        "serbian_cyrillic",
+        "serbian_latin",
+        "sinhala",
+        "somali",
+        "swahili",
+        "tamil",
+        "telugu",
+        "thai",
+        "tigrinya",
+        "turkish",
+        "ukrainian",
+        "urdu",
+        "uzbek",
+        "vietnamese",
+        "yoruba",
+    ]
+
+    instantiation_guide = """The languages supported for the XLSum dataset are:
```

Comment:
any chance we should put this in the description that comes with the dataset class?